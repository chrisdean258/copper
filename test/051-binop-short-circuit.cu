if false && { print("Should not print #1") false } {
	 print("Should not print #2")
}

if false && { print("Should not print #3") true } {
	 print("Should not print #4")
}

if true && { print("Should print #1") false } {
	 print("Should not print #5")
}

if true && { print("Should print #2") true } {
	 print("Should print #3")
}
