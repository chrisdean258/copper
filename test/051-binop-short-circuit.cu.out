Should print #1
Should print #2
Should print #3
